adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/validator/ResponseContentTypeRamlValidator.java

```diff
@@ -16,7 +16,7 @@
 public class ResponseContentTypeRamlValidator extends AbstractContentTypeRamlValidator {
 
     public ResponseContentTypeRamlValidator() {
-        super(ActionType.GET, "response type", "query");
+        super(ActionType.GET, "response type");
```

Comment:
Now set as constant internal to class, only used for validation description output.